examples/i_choose_you/src/main.rs

```diff
@@ -0,0 +1,101 @@
+use iced::widget::combo_box::{self, ComboBox};
+use iced::widget::{column, Container, Space, Text};
+use iced::{Alignment, Length, Sandbox, Settings};
+use strum::{EnumIter, IntoEnumIterator, IntoStaticStr};
+
+struct PokemonPicker {
+    pokemon: Pokemon,
+    // We store the state of the combo box
+    // that in turn contains the vector of choices
+    // because displaying the combo box requires
+    // the vector of choices to stay alive throughout the lifetime
+    // of the combo box.
+    choices: combo_box::State<Pokemon>,
+}
+
+#[derive(Debug, Clone, Copy)]
+enum Message {
+    PokemonSelected(Pokemon),
+}
+
+// `strum::EnumIter` derives `iter()` function for an enum, which is useful
+// to create a vector of choices.
+// `strum::IntoStaticStr` derives `Into<&'static str>` for an enum, which is useful
+// to display the enum variant names.
+#[derive(Debug, Default, Clone, Copy, EnumIter, IntoStaticStr)]
+enum Pokemon {
+    Bulbasaur,
+    Charmander,
+    Meowth,
+    #[default]
+    Pikachu,
+    Squirtle,
+}
+
+impl Sandbox for PokemonPicker {
+    type Message = Message;
+
+    fn new() -> Self {
+        PokemonPicker {
+            pokemon: Pokemon::default(),
+            choices: combo_box::State::new(Pokemon::iter().collect()),
```

Comment:
Here, we create [`combo_box::State`](https://docs.rs/iced/latest/iced/widget/combo_box/struct.State.html) using the iterator that we obtained using the `strum` crate.

Reply:
okay
then add it as dependency to that example